### DISCUSSION TITLE
Grafana pg_profile report and timeseries graphs

### DISCUSSION THREAD

**Discussion by hvanderland** [zubkov-andrei/pg_profile#77] [Ideas]
Attached a grafana dashboard to show pg_profile report using your code.
Maybe something to add to the product. 

Also create some dashboards with timeseries graphs to show the top SQL statements per execution time and execution calls and one for wait events.

[pg_profile_grafana_4.2_202309.zip](https://github.com/zubkov-andrei/pg_profile/files/12753902/pg_profile_grafana_4.2_202309.zip)

**Comment by zubkov-andrei** [comment]
Hi!
Oh, thank you, that's interesting. I'll take a look at it.

**Comment by hvanderland** [comment]
Andrei

Did you check the pg_profile_report dashboard ? This gives online view of the data captured by pg_profile.
Same data that is showing when generating the report.  This is a quick way of checking the different environments.
